cmd/contour/serve.go

```diff
@@ -88,6 +88,14 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 			return fmt.Errorf("cannot specify both %s and %s", "--contour-config", "-c/--config-path")
 		}
 
+		if ctx.contourConfigurationName != "" {
```

Comment:
I believe we can drop this now and just let the Get call fail if the user has specified the flag but doesn't have ContourConfiguration CRDs installed.

Reply:
Makes sense!